spec/horoshop_spec.rb

```diff
@@ -3,21 +3,73 @@
 require 'spec_helper'
 
 describe Horoshop do
+  let(:url) { 'http://api.horosop.com' }
+  let(:username) { 'valid_user' }
+  let(:password) { 'valid_password' }
+  let(:invalid_username) { 'invalid_user' }
+  let(:invalid_password) { 'invalid_password' }
+
   describe '#initialize' do
     subject { described_class.new(params) }
 
     context 'when all parameters passed' do
       let(:params) do
-        { url: 'http://api.horosop.com', username: 'user', password: '123456' }
+        { url: url, username: username, password: password }
       end
 
       it { is_expected.to be_a Horoshop }
```

Comment:
We should also check that token is received and change variables token and expiration time